### ISSUE TITLE
Length check returns 0

### ISSUE THREAD

**Issue by magnusskogen** [verbb/vizy#38]
**Description**
Adding the `| length` twig filter returns 0, even if the field contains a value.

**Steps to reproduce**
1. Add content to a field inside a Vizy block.
2. Check length in template: `{{ node.fieldName | length }}`

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.15
- Multi-site: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
